[
  {
    "cmd": [
      "python",
      "-u",
      "import sys; sys.exit(1)"
    ],
    "name": "Recipe failed. Reason: Fake failure",
    "~followup_annotations": [
      "step returned non-zero exit code: 1",
      "@@@STEP_LOG_LINE@python.inline@import sys; sys.exit(1)@@@",
      "@@@STEP_LOG_END@python.inline@@@",
      "@@@STEP_FAILURE@@@"
    ]
  },
  {
    "name": "$result",
    "reason": "Step('Recipe failed. Reason: Fake failure') failed with return_code 1",
    "recipe_result": null,
    "status_code": 1
  }
]